app/js/components/header/Header.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import logo from '../../../img/openmrs-with-title-small.png';
+
+export default () => (
+  <header>
+    <div className="logo">
+      <a href="../../">
```

Comment:
Where does this link navigate to? Home? Is there any need to use this relative path?

Reply:
Oh! That is not right. I have addressed it. Thank you for that observation.